recipes/graylog-illuminate/recipe.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../tools'
+
+class GraylogIlluminate < FPM::Cookery::Recipe
+  description 'Graylog Illuminate'
+
+  name     'graylog-illuminate'
+  version  data.version
+  revision data.revision
+  homepage 'https://graylog.org'
+  arch     'amd64'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer 'Graylog, Inc. <hello@graylog.org>'
+  vendor     'graylog'
+  license    data.license
+
+  config_files   Dir['graylog_illuminate_core/*'],
+                 Dir['graylog_illuminate_windows/*'],
+                 Dir['graylog_illuminate_okta/*'],
+                 Dir['graylog_illuminate_paloalto/*'],
+                 Dir['graylog_illuminate_o365/*'],
+                 Dir['illuminate_elastic_template/*']
+
+  fpm_attributes rpm_os: 'linux'
```

Comment:
Why do we need this? We don't use it in the forwarder, server or enterprise packages.

Reply:
That is used in `collector-sidecar`. It was copied over because I used that as the base for this. I am not even sure what it's for. Just extra metadata? I'm sure it can be deleted.

Reply:
Fixed.